.github/workflows/test_and_lint.yml

```
@@ -26,11 +26,31 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: EgorDm/gha-yarn-node-cache@v1
-      
+
       - name: Install dependencies
         run: yarn install
 
       - name: Run tests
         run: yarn test
-      
-      
+
+      # Prepare the Pull Request Payload artifact. If this fails, we
+      # we fail silently using the `continue-on-error` option. It's
+      # nice if this succeeds, but if it fails for any reason, it
+      # does not mean that our lint-test checks failed.
+      - name: Prepare Pull Request Payload artifact
+        id: prepare-artifact
+        if: always() && github.event_name == 'pull_request'
+        continue-on-error: true
+        run: cat $GITHUB_EVENT_PATH | jq '.pull_request' > pull_request_payload.json
```

This is probably an absolute path so it can be quoted. If it relies on globs to expand then it shouldn't be quoted. Setting `-e` to be safe.
```suggestion
        run: jq -e '.pull_request' "$GITHUB_EVENT_PATH" > pull_request_payload.json
```